### ISSUE TITLE
Allow to mute add-on's notifications

### ISSUE THREAD

**Issue by mrEDitor** [Drive4ik/simple-tab-groups#1064]
**Is your feature request related to a problem? Please describe.**
I'm pretty annoyed when i got spammed with notifications because of just moving tabs. E.g. I have a morning routine of opening things to do for couple of next hours and installed the add-on to keep "not really interesting things" as a Firefox tabs group instead of bookmarks folder. Each of dozen or two tabs makes sound when I "post-pone" it to another group

**Describe the solution you'd like**
It would be great if there are some settings for notifications, I would like to keep them, but switch the sound off. But I don't want to mute whole Firefox in Windows notifications, obviously

**Comment by biltongza** [comment]
Yep I just downloaded the addon and the notifications honestly make me question if it's worth it. I don't need to be notified that I just moved a tab to a new group because that's the button I just clicked on.
